src/responder/common/responder.h

```diff
@@ -108,6 +108,10 @@ struct resp_ctx {
     int domains_timeout;
     int client_idle_timeout;
 
+    time_t last_request_time;
```

Comment:
Hmm. When re-working the patches I ended up removing a method that would do this for us.
I've introduced a --dbus-activated option (similarly to --socket-activated) in order to solve this.